README.md

```diff
@@ -25,9 +25,7 @@ huge node quantities.
 `item_drop` is a nice addition to this annoying mod. You can prevent players
 from doing something serious by feeding them useless items.
 
-
 ## Licensing
 
-Code: CC0/WTFPL
-
+Code: CC0/WTFPL   
```

Comment:
The empty new line was intended to add a new line in the markdown rendered output. Please have a look at this file in a markdown editor.

Reply:
Changes reverted.